src/components/KeywordRadioBtn/KeywordRadioBtnGroup.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { useState } from "react";
+
+import styles from "./KeywordRadioBtnGroup.module.scss";
+
+export function KeywordRadioBtnGroup() {
+  const [keyword, setKeyword] = useState("name");
+  // const onOptionChange = ({ e }: { e: ChangeEvent }) => {
+  //   const element = e.target as HTMLInputElement;
+  //   setKeyword(element.value);
+  const onOptionChange = (e) => {
+    setKeyword(e.target.value);
+  };
+
+  return (
+    <div className="radioGroupWrapper">
+      <div className="radioWrapper">
+        <input
+          type="radio"
+          id="name"
+          value="name"
+          name="keyword"
+          checked={keyword === "name"}
+          onChange={onOptionChange}
+        />
+        <label htmlFor="name">name</label>
+      </div>
+      <div className="radioWrapper">
+        <input
+          type="radio"
+          id="city"
+          value="city"
+          name="keyword"
+          checked={keyword === "city"}
+          onChange={onOptionChange}
+        />
+        <label htmlFor="city">city</label>
+      </div>
+      <div className="radioWrapper">
+        <input
+          type="radio"
+          id="country"
+          value="country"
+          name="keyword"
+          checked={keyword === "country"}
+          onChange={onOptionChange}
+        />
+        <label htmlFor="country">country</label>
+      </div>
+      <div className="radioWrapper">
```

Comment:
`radioWrapper` is an non existent class. And should't it be `{styles.radioWrapper}` ?

Reply:
Ok, need to correct the syntax. 
Otherwise: "It will shortly. I left it in because I'm refactoring from an existing project which already contains this class and styles linked to it. It serves to remind me which classes, elements and styles (will) go together."
Is that bad practice?

Reply:
done, it exists now